src/main.rs

```diff
@@ -1,50 +1,14 @@
-use crate::actors::{delete_actor, get_actors, new_actor};
-use crate::characters::{assign_character, get_characters};
-use actix_web::{web, App, HttpServer};
 use dotenvy::dotenv;
-use sqlx::postgres::PgPoolOptions;
-use sqlx::PgPool;
+use play_reading_party::{configure_database, get_server, migrate_database};
+use std::net::TcpListener;
 
-mod actors;
-mod characters;
-
-pub struct AppState {
-    db: PgPool,
-}
-
-#[actix_web::main]
-async fn main() -> std::io::Result<()> {
+#[tokio::main]
+// Run the main server for the application
```

Comment:
I'd argue this comment doesn't add clarity and should just be removed since anyone working on a rust project will know `main()` runs the application.